src/main/eo/org/eolang/math/number.eo

```diff
@@ -30,6 +30,8 @@
 +version 0.0.0
 
 [n] > number
+  n > @
```

Comment:
@levBagryansky do we really need this addition?

Reply:
@includealex Yes, this allows to use `n`'s attributes in object `(number n)`. For instance, without it you can not call `(number 1).neg`. Check how `text` works:
https://github.com/objectionary/eo-strings/blob/master/src/main/eo/org/eolang/txt/text.eo